netket/hilbert/__init__.py

```diff
@@ -15,6 +15,8 @@
 from .abstract_hilbert import AbstractHilbert
 from .discrete_hilbert import DiscreteHilbert
 
+from .continuous_space_particles import Particles
```

Comment:
to be consistent with the other Hilbert spaces, I think this should be called ```Particle``` (singular) and not ```Particles``` (sorry for the previous suggestion, I guess I had suggested the plural before) 

In this way it is actually more natural to do things like 

```python
particles=(Particle(L=(10,10,10))*Spin(1/2))**N 
```
which is the typical use case, with explicit tensor powers, we should push for more and more in the future. 

Of course the argument N can be left/ should be left both for spin and particle, but that's just a convenient short cut, I would say...